Source/WebKit/GPUProcess/GPUProcess.h

```diff
@@ -144,6 +146,10 @@ class GPUProcess final : public AuxiliaryProcess, public ThreadSafeRefCounted<GP
     std::optional<WebCore::ProcessIdentity> immersiveModeProcessIdentity() const;
 #endif
 
+#if HAVE(AUDIT_TOKEN)
+    void setPresentingApplicationAuditToken(WebCore::ProcessIdentifier, WebCore::PageIdentifier, const std::optional<CoreIPCAuditToken>&);
```

Comment:
If its only caller is IPC and it is good to move the audit token, we could have it as a && instead of const&